airflow/lineage/__init__.py

```diff
@@ -36,7 +36,6 @@ def _get_backend():
 
     try:
         _backend_str = conf.get("lineage", "backend")
-        prepare_classpath()
```

Comment:
Just want to know why was it present if we can just remove it.

Reply:
See previous comment.